difflint/lint.py

```diff
@@ -1,56 +1,152 @@
 # Copyright 2015 Endless Mobile, Inc.
 
+import json
 import os.path
+import pep8
 from pkg_resources import resource_filename
+import pyflakes.api
 import shutil
 
 from .lint_output import LintOutput
-from .lint_python import PythonLintOutput
+from .python_reporters import PEP8TerseReporter, PyFlakesTerseReporter
+
+
+_ENABLED_LINTERS_CONFIG = resource_filename(__name__, 'data/.difflintrc')
+
+
+def _read_enabled_linters_config():
+    """Reads from the configuration file to determine which linters
+    should be used for which file extensions.
+
+    Inputs: None
+
+    Output: A dictionary of languages mapped to a dictionary of extensions 
+            and linters.
+
+    Example Output Format:
+
+    {
+        "javascript": {
+            "extensions": ["js"],
+            "linters": ["jscs", "jshint"]
+        },
+        "python": {
+            "extensions": ["py", "pyw"],
+            "linters": ["pep8", "pyflakes"]
+        }
+    }
+    """
+    config_dict = {}
```

Comment:
I don't think this assignment has any effect. Is there some reason you can't just `return json.load(config)` below?